torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2495,6 +2495,8 @@ def create_symbol(
         dynamic_dim: DimDynamic = DimDynamic.DUCK,
         constraint_dim: DimConstraint = None,  # NB: includes None
         positive: Optional[bool] = True,
+        *,
+        unspecified: bool = False,
```

Comment:
Needs some docs distinguishing it from positive. (TBH all of this needs some refactor)

Reply:
I can do some refactoring for it.